monai/metrics/meandice.py

```diff
@@ -112,6 +117,15 @@ def aggregate(
 
         # do metric reduction
         f, not_nans = do_metric_reduction(data, reduction or self.reduction)
+        if self.reduction == MetricReduction.MEAN_BATCH and self.return_with_label:
+            _f = {}
+            if isinstance(self.return_with_label, bool):
+                for i, v in enumerate(f):
+                    _f[f"label_{i}"] = round(v.item(), 4)
```

Comment:
it seems setting `include_background` to `True` or `False` will make the output key `label_0` correspond to the first or second channel? I think this could be explained in the docstring or changed to have a consistent `label_0`.

Reply:
Thanks for pointing it out, have updated the docstring and fixed `label_0` to be the background in the latest commit.